app/helpers/views_helper.rb

```diff
@@ -0,0 +1,9 @@
+module ViewsHelper
+  def get_headings(kase, correspondence_type)
+    if kase.current_state == "rejected" && correspondence_type.abbreviation == CorrespondenceType.offender_sar.abbreviation
+      t4c(kase, "", "cases.new.offender_sar.rejected", case_type: kase.type)
+    else
+      t4c(kase, "", "cases.new.offender_sar.sub_heading", case_type: kase.type)
```

Comment:
If you look at the original method that was being called:
```
t4c(@case, '', 'sub_heading', case_type: @case.pretty_type)
```
The argument to case_type is `@case.pretty_type`, so you need to use the same thing here in both case. `pretty_type` is a decorator method so you need to call `kase.decorate.pretty_type` to access it.

When t4c is called from within a view, the `t` method will look for the translation in the part of the translation file that relates to that page. The 2nd argument of the t4c method allows you to specify where that is e.g. "cases.new".